examples/vllm-llama-faiss-run.yaml

```
@@ -0,0 +1,80 @@
+  provider_id: vllm
+  model_type: llm
+  provider_model_id: null
+- model_id: sentence-transformers/all-mpnet-base-v2
+  metadata:
+      embedding_dimension: 768
+  model_type: embedding
+  provider_id: sentence-transformers
+  provider_model_id: /home/USER/embedding_models/all-mpnet-base-v2
+  
+providers:
+  inference:
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 4 but found 2

(indentation)